hub/core/lock.py

```diff
@@ -11,9 +11,50 @@
 from hub.util.path import get_path_from_storage
 from hub.util.threading import terminate_thread
 from hub.core.storage import StorageProvider
+from hub.constants import FIRST_COMMIT_ID
+
+
+def _get_lock_bytes() -> bytes:
```

Comment:
can we add this as tobytes and from buffer within the object for consistency with other existing objects?

Reply:
These are not serialize/deserialize methods for the lock objects.